src/app/caution/page.module.css

```
@@ -0,0 +1,168 @@
+.appContainer {
+  box-sizing: border-box;
+  margin-top: -1rem;
+  min-height: 100vh;
+  font-family: inherit;
+  width: 100%;
+  position: relative;
+  overflow-x: hidden;
+}
+
+.mainContent {
+  position: relative;
+  margin: 0 auto;
+  padding-top: var(--space-3xl);
```

`padding-top`のみで十分です
```suggestion
  padding-top: var(--space-3xl);
```